OJS.Workers.SubmissionProcessors/Workers/LocalSubmissionWorker.cs

```diff
@@ -8,8 +8,9 @@ public class LocalSubmissionWorker
     {
         private readonly int portNumber;
 
-        public LocalSubmissionWorker(int portNumber)
-            => this.portNumber = portNumber;
+        public LocalSubmissionWorker()
+        {
+        }
```

Comment:
We don't need to declare the constructor

Reply:
It was removed